e2e/optimism_test.go

```diff
@@ -215,7 +231,9 @@ func testOptimismGenericCommitment(t *testing.T, v2Enabled bool) {
 
 	optimism.sequencer.ActL2PipelineFull(ot)
```

Comment:
Does this enact the entire derivation pipeline using the first L1 batch or does it just catch up from a previous batch state checkpoint?

Reply:
I'm not sure, I just utilized what was already there for this test. Sam and I tried to go through and understand what precisely was happening in this test, but I didn't feel like we really arrived at a deep level of understanding.